client/src/style.css

```diff
@@ -1253,3 +1253,30 @@ sandbox header
     from {transform: rotate(0deg);}
     to {transform: rotate(359deg);}
  }
+
+
+/* Tooltip container */
+.tooltip {
+    position: relative;
+    display: inline-block;
```

Comment:
best to set `pointer-event: none` to minimize the chance that the invisible div would block other divs